addon/templates/components/polaris-pagination.hbs

```diff
@@ -1,39 +1,39 @@
-<button
-  data-test-prev-btn="true"
-  class="Polaris-Pagination__Button"
-  aria-label="Previous"
-  disabled={{isPreviousDisabled}}
-  onclick={{action onPrevious}}
-  onmouseup={{handleMouseUpByBlurring}}
+<nav
+  aria-label={{this.accessibilityLabel}}
+  class="Polaris-Pagination {{if @plain "Polaris-Pagination--plain"}}"
+  ...attributes
 >
-  {{polaris-icon source="arrow-left"}}
-</button>
+  <button
+    data-test-prev-btn="true"
+    class="Polaris-Pagination__Button"
+    aria-label="Previous"
+    disabled={{this.isPreviousDisabled}}
+    {{on "click" (or @onPrevious this.onPrevious)}}
+    {{on "mouseup" this.handleMouseUpByBlurring}}
+  >
+    <PolarisIcon @source="arrow-left" />
+  </button>
 
-<button
-  data-test-next-btn="true"
-  class="Polaris-Pagination__Button"
-  aria-label="Next"
-  disabled={{isNextDisabled}}
-  onclick={{action onNext}}
-  onmouseup={{handleMouseUpByBlurring}}
->
-  {{polaris-icon source="arrow-right"}}
-</button>
+  <button
+    data-test-next-btn="true"
+    class="Polaris-Pagination__Button"
+    aria-label="Next"
+    disabled={{this.isNextDisabled}}
+    onclick={{fn (or @onNext this.onNext)}}
```

Comment:
can we use modifiers here too? Ideally, we'd want to be consistent and use them everywhere instead of the old event bindings